src/app/app.component.html

```diff
@@ -1,9 +1,9 @@
 <!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
   <h1>
-    Nascar Fantasy Home Page
+    The Rockwell Cup
```

Comment:
Do not name the title page the Rockwell cup. That's tacky sir....

Reply:
fine